.sql/data.sql

```diff
@@ -0,0 +1,246 @@
+(4, 'Documents');
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `File`
+--
+
+CREATE TABLE `File` (
+  `id` int(11) NOT NULL,
+  `name_add` varchar(255) NOT NULL,
+  `name_view` varchar(255) DEFAULT NULL,
+  `name_hash` varchar(255) NOT NULL,
+  `subcategory_id` int(11) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `Format`
+--
+
+CREATE TABLE `Format` (
+  `id` int(11) NOT NULL,
+  `name` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+--
+-- Zrzut danych tabeli `Format`
+--
+
+INSERT INTO `Format` (`id`, `name`) VALUES
+(1, 'jpeg'),
+(2, 'png'),
+(3, 'svg'),
+(4, 'eps'),
+(5, 'txt'),
+(6, 'doc'),
+(7, 'ppt'),
+(8, 'xls');
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `MapFormat`
+--
+
+CREATE TABLE `MapFormat` (
+  `id` int(11) NOT NULL,
+  `file_id` int(11) DEFAULT NULL,
+  `format_id` int(11) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `MapTag`
+--
+
+CREATE TABLE `MapTag` (
+  `id` int(11) NOT NULL,
+  `file_id` int(11) DEFAULT NULL,
+  `tag_id` int(11) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `Subcategory`
+--
+
+CREATE TABLE `Subcategory` (
+  `id` int(11) NOT NULL,
+  `name` varchar(255) NOT NULL,
+  `category_id` int(11) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `Tag`
+--
+
+CREATE TABLE `Tag` (
+  `id` int(11) NOT NULL,
+  `name` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+--
+-- Indeksy dla zrzutów tabel
+--
+
+--
+-- Indexes for table `Category`
+--
+ALTER TABLE `Category`
+  ADD PRIMARY KEY (`id`);
+
+--
+-- Indexes for table `File`
+--
+ALTER TABLE `File`
+  ADD PRIMARY KEY (`id`),
+  ADD KEY `subcategory_id` (`subcategory_id`);
+
+--
+-- Indexes for table `Format`
+--
+ALTER TABLE `Format`
+  ADD PRIMARY KEY (`id`);
+
+--
+-- Indexes for table `MapFormat`
+--
+ALTER TABLE `MapFormat`
+  ADD PRIMARY KEY (`id`),
+  ADD KEY `file_id` (`file_id`),
+  ADD KEY `format_id` (`format_id`);
+
+--
+-- Indexes for table `MapTag`
+--
+ALTER TABLE `MapTag`
+  ADD PRIMARY KEY (`id`),
+  ADD KEY `file_id` (`file_id`),
+  ADD KEY `tag_id` (`tag_id`);
+
+--
+-- Indexes for table `Subcategory`
+--
+ALTER TABLE `Subcategory`
+  ADD PRIMARY KEY (`id`),
+  ADD KEY `category_id` (`category_id`);
+
+--
+-- Indexes for table `Tag`
+--
+ALTER TABLE `Tag`
+  ADD PRIMARY KEY (`id`);
+
+--
+-- AUTO_INCREMENT for dumped tables
+--
+
+--
+-- AUTO_INCREMENT dla tabeli `Category`
+--
+ALTER TABLE `Category`
+  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=5;
```

Comment:
AUTO_INCREMENT możesz wrzucić tak samo jak klucze bezpośrednio do CREATE TABLE ...
Jak będziesz robić encje to zerknij na: https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/basic-mapping.html#identifiers-primary-keys